Dockerfile

```diff
@@ -17,9 +17,10 @@ RUN             git config --global user.email "email@example.com"
 RUN             virtualenv -p /usr/bin/python3 /venv
 RUN             . /venv/bin/activate && pip install jupyter jupytext matplotlib tensorflow torch
 
-RUN             wget https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz
-RUN             tar -xzv < julia-1.3.1-linux-x86_64.tar.gz
-RUN             ln -s /julia-1.3.1/bin/julia /usr/bin/julia
+RUN             wget https://julialang-s3.julialang.org/bin/linux/x64/1.5/julia-1.5.3-linux-x86_64.tar.gz && \
```

Comment:
I can't speak on whether the version should be updated [someone more experienced with Gen needs to comment on that]. But I would probably parameterize the Julia version to make version changes quick and easy. Something like this:
```
# Find current Julia version on https://julialang.org/downloads/
ARG JULIA_VERSION_SHORT="1.5"
ARG JULIA_VERSION_FULL="${JULIA_VERSION_SHORT}.3"
ENV JULIA_INSTALLATION_PATH=/opt/julia

RUN wget https://julialang-s3.julialang.org/bin/linux/x64/${JULIA_VERSION_SHORT}/julia-${JULIA_VERSION_FULL}-linux-x86_64.tar.gz && \
    tar zxf julia-${JULIA_VERSION_FULL}-linux-x86_64.tar.gz && \
    mkdir -p "${JULIA_INSTALLATION_PATH}" && \
    mv julia-${JULIA_VERSION_FULL} "${JULIA_INSTALLATION_PATH}/" && \
    ln -fs "${JULIA_INSTALLATION_PATH}/julia-${JULIA_VERSION_FULL}/bin/julia" /usr/local/bin/ && \
    rm julia-${JULIA_VERSION_FULL}-linux-x86_64.tar.gz && \
    julia -e 'import Pkg; Pkg.add("IJulia")'
```
This should work in both the main image and the 20.04 image, so if you agree, you might want to adapt both. Sounds reasonable?

Reply:
that seems nice to me, and good point about the 20.04 image too.